test/jsdom.js

```diff
@@ -1,5 +1,6 @@
-var jsdom = require("jsdom");
+import * as jsdom from "jsdom";
 
-module.exports = function(html) {
+export default function(html) {
   return (new jsdom.JSDOM(html)).window.document;
```

Comment:
```suggestion
  const dom = new jsdom.JSDOM(html);
  global.document = dom.window.document
```

You need to make a global document if you want the tests to find the document. 

Change the code with the snippet at then the tests will find it. But you will need to change the code so that it calls the method and not initialize it as it is now.